src/org/usfirst/frc/team166/robot/subsystems/Drive.java

```diff
@@ -74,6 +74,7 @@
 
 	public Drive() {
 		initializeGear();
+		// setPIDConstants();
```

Comment:
Why add commented-out code? Either it should have been added in one commit, and then removed later on, in which case it's in the git history, or it wasn't worth adding in the first place.